src/components/course/CourseHeader.jsx

```diff
@@ -99,7 +107,7 @@ const CourseHeader = () => {
           <Col xs={12} lg={12}>
             {catalog.containsContentItems ? (
               <>
-                {enableReviewSection && <CourseReview />}
+                {enableReviewSection && isExperimentVariationA && <CourseReview />}
```

Comment:
How are we going to accomplish the second point mentioned in the description of ticket?